sdk/storage/azure-storage-files-datalake/test/datalake_file_client_test.cpp

```diff
@@ -570,6 +571,8 @@ namespace Azure { namespace Storage { namespace Test {
               AdlsGen2ConnectionString(), m_fileSystemName, objectName)
               .GetUri());
 
+      std::this_thread::sleep_for(std::chrono::seconds(30));
+
```

Comment:
This is not good.

Loop the result every 1 or 2 seconds, so that we can save some time. If we still cannot get expected result in 30 seconds, fail.

Reply:
I personally don't like doing multiple retries for a definitive result documented by the service. This may hide random occurring bugs. So I'll leave it as is. Please bring this up in our internal discussion if you still cannot accept it.

Reply:
I'm afraid I cannot accept this approach. I think we've set a horrible precedent here.